### ISSUE TITLE
Image with fit cover is getting stretched

### ISSUE THREAD

**Issue by Klabauterman** [DavBfr/dart_pdf#202]
**Describe the bug**
I would expect that if I place a 1:1 format image into a 10:3 Image windget with fit Box.cover to be placed centered into the widget so that it gets cut top and bottom.
What actually happens is that the full image is getting stretched to fill the 10:3 area.

**To Reproduce**
Code snippet to reproduce the behavior:
```dart
AspectRatio(
                aspectRatio: 10/3,
                 child: Image(image, fit: BoxFit.cover)
            )
```

**Comment by DavBfr** [comment]
You're right, this is not working.

**Comment by DavBfr** [comment]
Fixed in pdf version 1.4.1.
Don't hesitate to [Buy Me A Coffee](https://www.buymeacoffee.com/JORBmbw9h).
